### PR TITLE
Tech/upgrade everything, including corejs

### LINKED ISSUE #77: Upgrade to core-js 3
Let's upgrade to core-js 3

### PR DESCRIPTION
**Pull request by ndelangen**
fixes: #77 

This upgrade everything to latest and fixes deprecations

### CODE REVIEW

**Comment by ndelangen** [comment]
@xyc Would you want to review this?

**Review by xyc — APPROVED** [review]
Looks good, thanks! just curious why the registry change in yarn.lock?

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-      },
-    ],
-  },
+module.exports = async ({ config }) => {
+  console.log(config);
+  config.module.rules.push({
+    test: /\.(js|jsx)$/,
+    loaders: ['react-hot-loader', 'babel-loader'],
+    include: [path.join(__dirname, 'src')],
+    // include: [path.join(__dirname, 'src'), path.join(__dirname, 'example')]
```
Comment (wuweiweiwu) [inline .storybook/webpack.config.js]: nit: console log & comment

**Review by wuweiweiwu — APPROVED** [review]
wooooo!!

File: .storybook/webpack.config.js (hunk 1 above)
Comment (ndelangen) [reply .storybook/webpack.config.js]: I'll fix

**Comment by ndelangen** [comment]
> why the registry change in yarn.lock

bug, I'll fix it

**Comment by wmertens** [comment]
perfect timing, I just got `regeneratorRuntime is not defined` from the es build with my babel 7 stack, I'm thinking this PR might fix it. Any chance of a release?

**Pull request merged**